Extensions/Lighting/lightruntimeobject-pixi-renderer.js

```diff
@@ -17,10 +18,11 @@ gdjs.LightRuntimeObjectPixiRenderer = function (runtimeObject, runtimeScene) {
     objectColor[2] / 255,
   ];
 
-  /** @type {?PIXI.Texture} */
-  this._texture = runtimeObject.getPIXITexture();
-  this._center = new Float32Array([runtimeObject.x, runtimeObject.y]);
+  /** @type {string} */
```

Comment:
This should be `?string`. I'll change it in the next commit.

Reply:
Thanks :)

Reply:
Oops acutally no, this variable is holding either null or a PIXI.Texture (we're in the renderer). So `?PIXI.Texture` was valid.